.github/workflows/main-vmui.yml

```diff
@@ -0,0 +1,71 @@
+name: main-vmui
+
+on:
+  push:
+    branches:
+      - cluster
```

Comment:
There is no cluster branch in the https://github.com/VictoriaMetrics/VictoriaLogs/ repo.